features/support/capybara.rb

```diff
@@ -90,6 +90,7 @@ def is_a?(klass)
 Capybara.register_driver :chrome do |app|
   options = Selenium::WebDriver::Chrome::Options.new
   options.add_argument('--window-size=1280,2048')
+  options.add_argument('--disable-search-engine-choice-screen')
```

Comment:
What's this for?

Reply:
To disable this modal window when running cukes with chromedriver:
![image](https://github.com/user-attachments/assets/297e1c9e-02aa-4681-b13e-a1efee8f14f5)

Reply:
Interesting, thanks for the info.